### PR TITLE
Add documentation for --rebuild option

### PR DESCRIPTION
**Pull request by cfenoy**

### CODE REVIEW

File: docs/version-specific/eb_help.txt [hunk 1]
```diff
@@ -41,6 +41,7 @@
     -l, --logtostdout   Redirect main log to stdout (def False)
     -b BLOCKS, --only-blocks=BLOCKS
                         Only build listed blocks (type comma-separated list)
+    --rebuild           Rebuild software, even if module already exists (don't skip OS dependencies checks) (def False)
```
Comment (boegel) [inline docs/version-specific/eb_help.txt]: update the line for --force too?

File: docs/Using_the_EasyBuild_command_line.rst [hunk 2]
```diff
@@ -532,6 +542,7 @@ Note how the different status symbols denote distinct handling states by EasyBui
 * ``[ ]`` The build is not available, EasyBuild will deliver it
 * ``[x]`` The build is available, EasyBuild will skip building this module
 * ``[F]`` The build is available, however EasyBuild has been asked to force a rebuild and will do so
+* ``[R]`` The build is available, and the application will be rebuilt as request by rebuild flag
```
Comment (boegel) [inline docs/Using_the_EasyBuild_command_line.rst]: under the condition that https://github.com/hpcugent/easybuild-framework/pull/1435 is updated :)

Comment (boegel) [inline docs/Using_the_EasyBuild_command_line.rst]: replace "by rebuild flag" with "via `--rebuild`"

File: docs/Using_the_EasyBuild_command_line.rst [hunk 3]
```diff
@@ -532,6 +542,7 @@ Note how the different status symbols denote distinct handling states by EasyBui
 * ``[ ]`` The build is not available, EasyBuild will deliver it
 * ``[x]`` The build is available, EasyBuild will skip building this module
 * ``[F]`` The build is available, however EasyBuild has been asked to force a rebuild and will do so
```
Comment (boegel) [inline docs/Using_the_EasyBuild_command_line.rst]: mention "`--force`" here?

File: docs/Using_the_EasyBuild_command_line.rst [hunk 4]
```diff
@@ -260,6 +260,16 @@ Use ``eb --force/-f`` to force the reinstallation of a given easyconfig/module.
 .. tip:: Combine ``--force`` with ``--dry-run`` to get a good view on which installations will be forced.
    (cfr. :ref:`get_an_overview`)
 
+Rebuild installation, ``--rebuild``
```
Comment (boegel) [inline docs/Using_the_EasyBuild_command_line.rst]: wouldn't it make sense to collapse the sections for `--force` and `--rebuild` instead? this is more or less a copy-paste, not a big fan of that

**Comment by cfenoy** [comment]
I'll review the copy paste tomorrow and try to merge both sections

**Comment by boegel** [comment]
@cfenoy: ping? is it tomorrow yet? ;)

**Pull request merged**